File: 3581-the-two-sneaky-numbers-of-digitville/the-two-sneaky-numbers-of-digitville.cs
public class Solution {
    public int[] GetSneakyNumbers(int[] nums) {
        List<int> number = new List<int>();

        for(int i = 0;i<nums.Length;i++){
            for(int j= i+1;j<nums.Length;j++){
                if(nums[i]==nums[j]){
                    if(!number.Contains(nums[i])){
                        number.Add(nums[i]);
                    }
                }
            }
        }
    return number.ToArray();
    }
}

File: 3581-the-two-sneaky-numbers-of-digitville/the-two-sneaky-numbers-of-digitville.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var getSneakyNumbers = function(nums) {
let arr = []
for(let i = 0;i<nums.length;i++){
    for(let j=i+1;j<nums.length;j++){
        if(nums[i]===nums[j]){
            if(!arr.includes(nums[i])){
                arr.push(nums[i])
            }
        }
    }
}
    return arr

};